src/activityLog/ActivityLogsTreeItem.ts

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Activity, AzExtParentTreeItem, AzExtTreeItem, callWithTelemetryAndErrorHandling, IActionContext } from '@microsoft/vscode-azext-utils';
+import { localize } from '../utils/localize';
+import { ActivityTreeItem } from './ActivityTreeItem';
+
+export class ActivityLogTreeItem extends AzExtParentTreeItem {
+    public label: string = localize('activityLog', 'Activity Log');
+    public contextValue: string = 'azureActivityLog';
+
+    private activityTreeItems: Record<string, ActivityTreeItem> = {};
+
+    public constructor() {
+        super(undefined);
+    }
+
+    public async addActivity(activity: Activity): Promise<void> {
+        await callWithTelemetryAndErrorHandling('registerActivity', async (context: IActionContext) => {
+            this.activityTreeItems[activity.id] = new ActivityTreeItem(this, activity);
+            await this.refresh(context);
+        });
+    }
+
+    public async clearActivities(context: IActionContext): Promise<void> {
```

Comment:
We should probably have a way to clear just one or all the finished activies.

Reply:
I like that idea.